docs/superpowers/plans/2026-05-19-composer-extensions-plan.md

```
@@ -0,0 +1,1899 @@
+        return new Content(
+            text: 'mail.mention-notification',
+        );
+    }
+}
+```
+
+- [ ] **Step 2: Create the Blade view**
+
+Create `resources/views/mail/mention-notification.blade.php`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)